templates_ark/example_pixel_clustering.ipynb

```diff
@@ -767,11 +767,38 @@
     "with open(os.path.join(base_dir, pixel_output_dir, '%s_cell_clustering_params.json' % pixel_cluster_prefix), 'w') as fh:\n",
```

Comment:
<p>This should link to a section in the readme which describes why we use mantis. If you make the links I'll fill in the text. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/641/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>